inc/compatibility/elementor.php

```diff
@@ -456,6 +456,24 @@ public static function is_elementor_template( $elementor_template_type ) {
 		return self::$cache_cp_has_template[ $location ];
 	}
 
+	/**
+	 * Detect if the a page is using the checkout widget.
+	 */
+	public static function is_elementor_checkout() {
+		if ( ! class_exists( '\ElementorPro\Plugin', false ) ) {
+			return false;
+		}
+
+		$is_elementor_checkout = false;
+		$page_id               = get_the_ID();
+		$elementor_data        = get_post_meta( $page_id, '_elementor_data', true );
+		if ( ! empty( $elementor_data ) && is_string( $elementor_data ) && strpos( $elementor_data, 'woocommerce-checkout-page' ) ) {
+			$is_elementor_checkout = true;
```

Comment:
just a recommendation, not necessarily one for that case (since the it's serialized array and the searched data ins not located at first):

```suggestion
		$elementor_data        = get_post_meta( $page_id, '_elementor_data', true );
		if ( ! empty( $elementor_data ) && is_string( $elementor_data ) && strpos( $elementor_data, 'woocommerce-checkout-page' )!==false ) {
			$is_elementor_checkout = true;
```

As you know `strpos` returns the position of the founded string. If the string is positioned at the beginning of the string; it returns 0. That causes if the condition is evaluated as false.

an example: https://onlinephp.io?s=s7EvyCjg5eLlKkssik8pzS3QKC4pKsgv1lBPTEpWSElV11EAsdQ1Na1BqjLTcMpX83JxpiZn5IP5Cmn5pXkp6kA9tTi1AUlkXSmpSJoA&v=7.4.33%2C7.2.34%2C7.1.33%2C7.0.33%2C8.0.26